framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -1635,6 +1641,15 @@ def genCmd(self, cmdinfo, name, alias):
 
         # Create the declaration for the function prototype
         proto = cmdinfo.elem.find('proto')
+
+        # If there are error codes, add them to the list for this command
+        errorcodes = cmdinfo.elem.get('errorcodes')
+        if errorcodes is not None:
+            errors = []
```

Comment:
`self.all_cmd_errors[name] = errorcodes.split(',')` can be used in place of these four lines of code if you'd like.